next-cloudinary/src/loaders/cloudinary-loader.js

```diff
@@ -38,23 +39,25 @@ export function cloudinaryLoader(defaultOptions, cldOptions) {
     quality: 'auto',
     ...defaultOptions
   };
+  let publicId = getPublicId(options.src);
 
-  const cldImage = cld.image(options.src);
+  const cldImage = cld.image(publicId);
 
   transformationPlugins.forEach(({ plugin }) => {
-    const { options: pluginOptions } = plugin({
-      cldImage,
-      options,
-      cldOptions
-    }) || {};
+    const { options: pluginOptions } =
+      plugin({
```

Comment:
would you mind removing all of the formatting changes? 😬  guessing it was a linter